eisbuk-admin/src/components/slots/SlotListByDay/SlotsDay.tsx

```diff
@@ -135,7 +138,22 @@ const SlotsDay: React.FC<SlotsDayProps> = ({
 
   return (
     <>
-      {view.startsWith("slots") ? (
+      {view === CustomerRoute.Calendar ? (
+        <Grid className={classes.bookingsListContainer} container spacing={3}>
```

Comment:
This is a bit simpler in my opinion: Since you only have three options and you want to check if its one of the two, it's clearer to check if it's the remaining one, and if it isn't, by logic, it's automatically one of the two.

Reply:
Yeah, I know. That was my original reasoning but for some reason it broke the admin slots view...I'll merge this PR and check if it works though.